src/language/__tests__/printer-test.js

```diff
@@ -190,7 +190,7 @@ describe('Printer: Query document', () => {
 
       fragment frag on Friend {
         foo(size: $size, bar: $b, obj: {key: "value", block: """
-          block string uses \"""
+          block string uses \\"""
```

Comment:
Instead of changing this to `\\`, could you use `String.raw` as discussed in #1202? This test case intends to show the actual GraphQL string written, so avoiding double-encoding is intentional.